src/main/java/gg/projecteden/nexus/features/commands/BoopCommand.java

```diff
@@ -53,6 +54,9 @@ public void run(Player booper, Player booped, String message, boolean anonymous)
 		if (isSelf(booped))
 			error("You cannot boop yourself!");
 
+		if (MuteMenuUser.hasMuted(booped, MuteMenuCommand.MuteMenuProvider.MuteMenuItem.MESSAGES))
```

Comment:
Couple things:
1. Can you make this a separate mute menu item? 
2. Make sure it skips players in boop all as well